config/default.php

```diff
@@ -205,6 +205,13 @@
 $raidApiKey = '';                                                   // Raid API Key, '' to deny access
 $sendRaidData = false;                                              // Send Raid data, false to only send gym data
 
+//-----------------------------------------------------
+// Areas
+//-----------------------------------------------------
+
+$noAreas = false;
```

Comment:
This should be `true` because the array is empty and it would give an empty dropdown.

Reply:
fixed